.ci/build-docker-images.groovy

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'linux && immutable' }
+  environment {
+    REPO = 'beats'
+    BASE_DIR = "src/github.com/elastic/beats"
+    DOCKER_REGISTRY = 'docker.elastic.co'
+    DOCKER_REGISTRY_SECRET = 'secret/observability-team/ci/docker-registry/prod'
+    GOPATH = "${env.WORKSPACE}"
+    HOME = "${env.WORKSPACE}"
+    NOTIFY_TO = credentials('notify-to')
+    PATH = "${env.GOPATH}/bin:${env.PATH}"
+    PIPELINE_LOG_LEVEL='INFO'
+  }
+  options {
+    timeout(time: 1, unit: 'HOURS')
+    buildDiscarder(logRotator(numToKeepStr: '20', artifactNumToKeepStr: '20'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+    rateLimitBuilds(throttle: [count: 60, durationName: 'hour', userBoost: true])
+    quietPeriod(10)
+  }
+  triggers {
+    cron '@daily'
+  }
+  parameters {
+    booleanParam(name: "RELEASE_TEST_IMAGES", defaultValue: "true", description: "If it's needed to build & push Beats' test images")
+  }
+  stages {
+    stage('Checkout') {
+      steps {
+        dir("${BASE_DIR}"){
+          git("https://github.com/elastic/beats.git")
+        }
+        script {
+          dir("${BASE_DIR}"){
+            env.GO_VERSION = readFile(".go-version").trim()
+          }
+        }
```

Comment:
```suggestion
        setEnvVar('GO_VERSION', readFile("${BASE_DIR}/.go-version").trim())
```

Reply:
This is present in the old pipeline, maybe we need to triage the pipelines in there looking for new step versions

Reply:
I will follow on this and other changes once we have this merged and running. Thanks!